tools/installer/requirements.txt

```diff
@@ -1,3 +1,5 @@
+git+git://github.com/exaile/pywin32-ctypes.git@pyinstaller#egg=pywin32-ctypes
+git+git://github.com/dangmai/pyinstaller.git@msys2#egg=PyInstaller
 musicbrainzngs
 mutagen==1.31
```

Comment:
I just kept the requirement that we have currently, no opposition to bump it at all. Although I do think pinning our dependencies to a particular version is a good idea so that we can have a reproducible build. On the other hand we're running pacman to get the binary dependencies for the Windows build without pinning anyway, so maybe what we actually need a good integration test suite to detect regression (but ain't no body got time for that 😛 )